tools/crm_mon_print.c

```diff
@@ -2037,6 +2037,7 @@ print_failed_stonith_actions(mon_state_t *state, stonith_history_t *history, uns
     }
 
     /* Print each failed stonith action */
+    history = stonith__sort_history(history);
```

Comment:
It looks like stonith_history is already sorted (in mon_refresh_display()) by the time it gets to print_status() etc.